src/data/thermal-conductivity-constants.ts

```diff
@@ -16,8 +16,374 @@ export const DIPPRThermalConductivityDictionary: Record<
+    thermalConductivityAtMinimumTemperature: 0.2116,
+    thermalConductivityAtMaximumTemperature: 0.1413,
+  },
+  Acetonitrile: {
+    name: 'Acetonitrile',
+    formula: 'C2H3N',
+    CAS: ' 75-05-8',
+    molecularWeight: 41.0519,
+    C1: 0.30755,
+    C2: -0.000402,
+    minimumTemperature: 229.32,
+    maximumTemperature: 354.81,
+    thermalConductivityAtMinimumTemperature: 0.2154,
+    thermalConductivityAtMaximumTemperature: 0.1649,
+  },
+  Acetylene: {
+    name: 'Acetylene',
+    formula: 'C2H2',
+    CAS: '74-86-2',
+    molecularWeight: 26.03728,
+    C1: 0.33363,
+    C2: -0.00083655,
+    minimumTemperature: 192.4,
+    maximumTemperature: 250,
+    thermalConductivityAtMinimumTemperature: 0.1727,
+    thermalConductivityAtMaximumTemperature: 0.1245,
+  },
+  Acrolein: {
+    name: 'Acrolein',
+    formula: 'C3H4O',
+    CAS: '107-02-8',
+    molecularWeight: 56.06326,
+    C1: 0.2703,
+    C2: -0.0003764,
+    minimumTemperature: 185.45,
+    maximumTemperature: 325.84,
+    thermalConductivityAtMinimumTemperature: 0.2005,
+    thermalConductivityAtMaximumTemperature: 0.1477,
+  },
+  AcrylicAcid: {
+    name: 'Acrylic acid',
+    formula: 'C3H4O2',
+    CAS: '79-10-7',
+    molecularWeight: 72.06266,
+    C1: 0.2441,
+    C2: -0.0002904,
+    minimumTemperature: 286.15,
+    maximumTemperature: 484.5,
+    thermalConductivityAtMinimumTemperature: 0.161,
+    thermalConductivityAtMaximumTemperature: 0.1034,
+  },
+  Acrylonitrile: {
+    name: 'Acrylonitrile',
+    formula: 'C3H3N',
+    CAS: '107-13-1',
+    molecularWeight: 53.0626,
+    C1: 0.30751,
+    C2: -0.000487,
+    minimumTemperature: 189.63,
+    maximumTemperature: 350.45,
+    thermalConductivityAtMinimumTemperature: 0.2152,
+    thermalConductivityAtMaximumTemperature: 0.1368,
+  },
+  Air: {
+    name: 'Air',
+    formula: 'Mixture',
+    CAS: '132259-10-0',
+    molecularWeight: 28.96,
+    C1: 0.28472,
+    C2: -0.0017393,
+    minimumTemperature: 75,
+    maximumTemperature: 125,
+    thermalConductivityAtMinimumTemperature: 0.1543,
+    thermalConductivityAtMaximumTemperature: 0.0673,
+  },
+  Ammonia: {
+    name: 'Ammonia',
+    formula: 'H3N',
+    CAS: '7664-41-7',
+    molecularWeight: 17.03052,
+    C1: 1.169,
+    C2: -0.002314,
+    minimumTemperature: 195.41,
+    maximumTemperature: 400.05,
+    thermalConductivityAtMinimumTemperature: 0.7168,
+    thermalConductivityAtMaximumTemperature: 0.2433,
+  },
+  Anisole: {
+    name: 'Anisole',
+    formula: 'C7H8O',
+    CAS: '100-66-3',
+    molecularWeight: 108.13782,
+    C1: 0.23494,
+    C2: -0.00026477,
+    minimumTemperature: 235.65,
+    maximumTemperature: 512.5,
+    thermalConductivityAtMinimumTemperature: 0.1725,
+    thermalConductivityAtMaximumTemperature: 0.0993,
+  },
+  Argon: {
+    name: 'Argon',
+    formula: 'Ar',
+    CAS: '7440-37-1',
+    molecularWeight: 39.948,
+    C1: 0.1819,
+    C2: -0.0003176,
+    C3: -0.00000411,
+    C4: NaN,
+    C5: NaN,
```

Comment:
Should be removed

Reply:
@sumitnegi7  appove this review , merging is blocked due to this